cmd/gke-exec-auth-plugin/request.go

```diff
@@ -130,13 +131,15 @@ func processCSR(client clientset.Interface, privateKeyData []byte, hostname stri
 	csrData = append(csrData, attestData...)
 	klog.Info("added TPM attestation")
 
-	// TODO(liggitt): only add this key usage for RSA private keys once the minimum supported control plane is 1.25.
-	// see https://issue.k8s.io/109077
```

Comment:
ETA for this to be able to merge is ~2024 per EOL support timeline for 1.24 in https://cloud.google.com/kubernetes-engine/docs/release-schedule

Reply:
/hold

Reply:
Got it.